src/metrics_validator.py

```diff
@@ -0,0 +1,39 @@
+"""Validator for Hubble Metrics configuration values."""
+from typing import List
+
+from pydantic import BaseModel, validator
+
+
+class HubbleMetrics(BaseModel):
+    """Class to validate the values of the Hubble Metrics provided by the user."""
+
+    metrics: List[str]
+
+    @property
+    def allowed_values(self):
+        """Allowed metrics values for Hubble."""
+        return
```

Comment:
Sorry, I attempted to use a property instead of a dictionary in the validator function. However, it appears that the validator function is unable to utilize the property.